python/ray/_private/runtime_env/physical_mode_execution_utils.py

```diff
@@ -0,0 +1,177 @@
+"""A few util functions for physical mode execution.
+NOTE:
+1. It should be only used under linux environment; for other platforms, all functions
+are no-op.
+2. It assumes cgroup v2 has been mounted.
+"""
+# TODO(hjiang):
+# 1. Consider whether we need to support V1.
+# 2. Check the existence for cgroup V2.
+# 3. Add error handling; for now just use boolean return to differentiate success or
+# failure. Better to log the error.
+
+from ray._private.runtime_env import physical_mode_context
+
+import sys
+import os
+import shutil
+
+# There're two types of memory cgroup constraints:
+# 1. For those with limit memory capped, they will be created a dedicated cgroup;
+# 2. For those without limit memory specified, they will be added to the default memory
+# cgroup.
+DEFAULT_MEM_CGROUP_UUID = "default_cgroup_memory_uuid"
```

Comment:
IMO, we should not have memory in the cgroup name since cgroup v2 unifies all the resources, how about renaming to `default_ray_worker_cgroup`?

Reply:
Updated.